mount/temp.go

```diff
@@ -58,6 +58,24 @@ func WithTempMount(ctx context.Context, mounts []Mount, f func(root string) erro
 			}
 		}
 	}()
+
+	// The volatile option of overlayfs doesn't allow to mount again using the
+	// same upper / work dirs. Since it's a temp mount, avoid using that
+	// option here.
+	// TODO: Make this logic conditional once the kernel supports reusing
+	// overlayfs volatile mounts.
```

Comment:
It seems that the function doc comment needs to be updated to add the explanation.

Reply:
@ktock what do you mean by function doc comment?

Reply:
I meant the comment for the function itself: https://github.com/containerd/containerd/blob/c9505370f8b2459a30bb33d64c7cf1e0e4556b5d/mount/temp.go#L29-L32

Reply:
Hi @KodieGlosserIBM Please check out @ktock's comment. The comment should be for WithTempMount. And please squash into one commit. Thanks

Reply:
done!